pandas/__init__.py

```diff
@@ -36,6 +36,9 @@
 
 from datetime import datetime
 
+from pandas._config import (get_option, set_option, reset_option,
```

Comment:
how did these get in here now? (Ithink it is explicit and ok), but just wondering

Reply:
These used to be a "*" import, now made explicit.